docs/assets/templates/notebooks/template-small.dockstore.yml

```diff
@@ -4,12 +4,10 @@ notebooks:
     language: <PYTHON | R | JAVASCRIPT | SCALA | JULIA | OTHER>
     path: <String>
     kernel: <String>
-    otherFiles: <String Array>
-    testParameterFiles: <String Array>
+    otherFiles: <String Array>  # Must be a YAML array of strings
+    testParameterFiles: <String Array>  # Must be a YAML array of strings
```

Comment:
Do notebooks have test parameter files? The big template doesn't show it

Reply:
> Do notebooks have test parameter files? The big template doesn't show it

At present, we allow them to be specified, although we're not sure what they mean yet.